### ISSUE TITLE
[Bug]: MetaMask-beta - transaction with the low gas fails with the error: •Error in smart transaction publish hook (Error#292) • Error#292: Transaction does not have a transaction hash, there was a problem

### ISSUE THREAD

**Issue by sleepytanya** [MetaMask/metamask-extension#25845]
### Describe the bug

Transaction with the Low gas very often fails with the error.

### Expected behavior

-

### Screenshots/Recordings

<img width="651" alt="Screenshot 2024-07-15 at 21 04 51" src="https://github.com/user-attachments/assets/db274583-94ab-49c4-98d1-aa9caa3fea2a">

<img width="389" alt="Screenshot 2024-07-15 at 21 04 23" src="https://github.com/user-attachments/assets/d660c0c3-70ca-4c30-b296-c2b38d038016">

<img width="558" alt="Screenshot 2024-07-15 at 21 05 00" src="https://github.com/user-attachments/assets/3f791a5b-432c-43c9-9534-ae61e2db8db5">

### Steps to reproduce

1. Start smart transaction
2. Select Low in the gas settings

### Version

v12.0.0

### Build type

Beta

### Browser

Chrome

### Operating system

MacOS

**Comment by klejeune** [comment]
This issue has been acknowledged, but it is not a current priority. Closing for now, but feel free to reopen if needed in the future.

**Comment by abheektripathy** [comment]
hey @klejeune still facing this error on prod, even though very rare

added a 20% extra fee + the estimated fee as a quick fix, but will this be worked upon in the future as to have a more permanent solution?

**Comment by pumpelchen** [comment]
This problem only happens on Chrome, and not on edge. On Chrome the transaction is not even posted, at east i dont see it on the block explorer (https://etherscan.io/).
I dont't no why this is not fixed, even if there is a real workaround now (paying more gas fees as needed is no workaround for me). So this bug forces me to use edge instead of chrome now for everything.

**Comment by sleepytanya** [comment]
The issue is present in 12.11.0:

**Comment by httpJunkie** [comment]
@rarquevaux is working on a backend fix for this issue and it can be found here:
https://github.com/consensys-vertical-apps/va-mmcx-transaction-api/pull/76

TLDR: [Video of Fix](https://www.loom.com/share/8ea48cb627764c818a107e0f312f9491?sid=72f29573-e6cf-42f9-87af-95aad485ae9e)

**Comment by dan437** [comment]
It appears this issue has been resolved according to the previous comment, so I'm closing this ticket.
